specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2697,6 +3120,142 @@
       ],
       "description": "Parameter required for creating a linked server to redis cache."
     },
+    "RedisCacheAccessPolicyCreateUpdateParameters": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RedisCacheAccessPolicyProperties",
+          "description": "Properties required to create an access policy."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "Parameters required for creating an access policy for the redis cache"
+    },
+    "RedisCacheAccessPolicyGetPutResponse": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of access policy",
+          "readOnly": true
```

Comment:
It should be enum with set of values.

Reply:
Added enum for access policy and access policy assignments provisioning states and access policy types